### PR TITLE
Adding reference to React Native for Web Starter - fix #146

### LINKED ISSUE #146: [Documentation] Adding a reference to React Native for Web Starter
Hi everyone,

As I was trying out React Native for Web, I figure it would be useful to create and maintain a Starter kit, as lean as possible.

React Native for Web Starter aims at:
- Providing the leanest code base to bootstrap a React Native for Web project.
- Following the standard React Native project structure close so someone becoming familiar with React Native may quickly find its way in a RN4Web project, and vice-versa.

Feel free to check it out [here](https://github.com/grabcode/react-native-web)

Cheers

### PR DESCRIPTION
**Pull request by grabcode**
### [React Native for Web Starter](https://github.com/grabcode/react-native-web-starter)
Light change to the `readme.md`, offering another way to start a fresh React Native for Web project.

Fixing #146

##### Update
Fixing #148: I've added support to [React Storybook](https://github.com/kadirahq/react-storybook)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ npm install --save react@0.14 react-dom@0.14 react-native-web
 
 Read the [Client and Server rendering](docs/guides/rendering.md) guide.
 
+## React Native Web Starter
+
+You may also start a fresh React Native Web project from this starter. It provides a lean and familiar setup as it follows a standard React Native project structure.
+
+[>> React Native for Web Starter](https://github.com/grabcode/react-native-web)
```
Comment (MonirAbuHilal) [inline README.md]: The link seems incorrect, should it be "[https://github.com/grabcode/react-native-web-starter](https://github.com/grabcode/react-native-web-starter)" ?

Comment (grabcode) [inline README.md]: You are right @MonirAbuHilal - this is fixed now.

**Comment by grabcode** [comment]
@necolas what could we do to get more momentum over React Native Web?

**Comment by MoOx** [comment]
Maybe a support from some react-native contributors? See @vjeux comment here https://github.com/taobaofed/react-web/issues/87#issuecomment-222381434

**Comment by necolas** [comment]
Thanks. Applied in d53d1e6e56bcb21dc1b53c10f2ef96d2a6ea8c9d

**Pull request closed without merging**